_temp/server.pem

```diff
@@ -0,0 +1,31 @@
+-----BEGIN CERTIFICATE-----
+MIIB7zCCAVgCCQCjgczvtmS2FTANBgkqhkiG9w0BAQsFADA8MQswCQYDVQQGEwJL
+WTETMBEGA1UECAwKU29tZS1TdGF0ZTEYMBYGA1UECgwPVXJzdWxhJ3MgRG9udXRz
+MB4XDTE3MTAyNDE5MjkxMloXDTIwMDgxMzE5MjkxMlowPDELMAkGA1UEBhMCS1kx
+EzARBgNVBAgMClNvbWUtU3RhdGUxGDAWBgNVBAoMD1Vyc3VsYSdzIERvbnV0czCB
+nzANBgkqhkiG9w0BAQEFAAOBjQAwgYkCgYEA5J8ICNyZNzDD26VC42ipWz8UE+pp
+RtVtKW1ivejYsWSZfoA4pTyEXMOP60TKpikkJzWW0iAk7hL3fg+c8AWS0tB2hNXp
+az78kDKd2QoeisB4olZH8irJ+9Zmj66R8JhBmjlggkO9ysiAxIh15/78Feb4YrL5
+leGB7cuQJgYeA/MCAwEAATANBgkqhkiG9w0BAQsFAAOBgQC007sMgfmbGTOEUix0
+pqPw27MJJiMQGhVTvlsLiOXd/8ztlusKysRY+DeAi4kts8MIl6VQg39lZ8OgmvFU
+NB8xP/GjGcaiF8+pC2PvYFSIWQcPsalmonkt3BivbfU2Oqdoq62L5lYnT5QAzlk6
+u5NP9PK4HXXdsXAJY4JBJZ62jg==
+-----END CERTIFICATE-----
+-----BEGIN RSA PRIVATE KEY-----
+Proc-Type: 4,ENCRYPTED
```

Comment:
Omg, how did you generate it?)

Anyway, it absolutely doesn't matter for the code. But try the following:
```
openssl ecparam -genkey -name secp256k1 -out server.key
openssl req -new -key server.key -out server.csr
```
This way, you produce keys with exact same curve as Ethereum uses (and it will be ECDSA, not RSA)